### PR TITLE
[RW-4723][Risk=no] Fix eRA commons linking

### PR DESCRIPTION
**Pull request by blrubenstein**
Description:
This fixes the eRA commons redirect to follow up by setting the task to completed. In doing this, I realized there was a loading step, so I added a loading spinner to the button. I talked to Lou about the styling on that, and we decided to move all the icons to the right of the button, because if the spinner was on the left, it would push the text when it appeared. The button is disabled but still primary while loading per Lou's request. This has been confirmed with Karthik
Here is a screenshot while loading:
Screen Shot 2020-04-13 at 4 31 41 PM

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [x] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [x] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

**Comment by blrubenstein** [comment]
Unsure if it is worth adding a unit test for this. Let me know if you think so.

File: ui/src/app/pages/homepage/homepage.tsx [hunk 1]
```diff
@@ -65,27 +65,36 @@ export const styles = reactStyles({
...
-    firstVisit: boolean,
-    firstVisitTraining: boolean,
-    quickTour: boolean,
-    trainingCompleted: boolean,
-    twoFactorAuthCompleted: boolean,
-    videoOpen: boolean,
-    videoLink: string,
-    userHasWorkspaces: boolean | null
-  }> {
+interface Props {
```
Comment (als364) [inline ui/src/app/pages/homepage/homepage.tsx:68]: Thanks for breaking this out.

File: ui/src/app/pages/homepage/homepage.tsx [hunk 2]
```diff
@@ -109,6 +119,7 @@ export const Homepage = withUserProfile()(class extends React.Component<
     this.checkWorkspaces();
     this.validateNihToken();
     this.callProfile();
+    console.log('component did mount');
```
Comment (als364) [inline ui/src/app/pages/homepage/homepage.tsx:122]: extraneous

File: ui/src/app/pages/homepage/homepage.tsx [hunk 3]
```diff
@@ -124,9 +135,14 @@ export const Homepage = withUserProfile()(class extends React.Component<
 
   async validateNihToken() {
     const token = (new URL(window.location.href)).searchParams.get('token');
+    console.log(token);
```
Comment (als364) [inline ui/src/app/pages/homepage/homepage.tsx:138]: extraneous

**Pull request merged**